Providers/Modules/Plugins/Common/plugin/out_oms_changetracking_file.rb

```diff
@@ -0,0 +1,530 @@
+module Fluent
+
+  class OutChangetrackingFile < BufferedOutput
+
+    Plugin.register_output('out_oms_oms.changetracking.file', self)
```

Comment:
I've never seen a plugin registered with a name that has periods in it; I would advise you stick to convention and name this "out_oms_changetracking_file"